### ISSUE TITLE
fail to save or download .pptx as evidence

### ISSUE THREAD

**Issue by schmitz-ens** [intuitem/ciso-assistant-community#304]
When I add an evidence (Compliance/Evidence), it works with PDF, but not with PPTX. I get : 

```
frontend  | TypeError: Cannot read properties of undefined (reading '0')
frontend  |     at GET (file:///app/build/server/chunks/_server.ts-Bf9Kumuo.js:87:63)
frontend  |     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
frontend  |     at async render_endpoint (file:///app/build/server/index.js:1299:20)
frontend  |     at async resolve2 (file:///app/build/server/index.js:3816:22)
frontend  |     at async Object.handle (file:///app/build/server/chunks/hooks.server-DSThBFNA.js:50:10)
frontend  |     at async respond (file:///app/build/server/index.js:3710:22)
frontend  |     at async Array.ssr (file:///app/build/handler.js:1243:3)

```

On UI, file seems to be saved, but I get a 400 when I try to fetch it.

Client :

OS: Linux
Browser : Chrome 122.0.6261.128

Server :

Version:v1.1.0 
Build:12e6bf2

Launched via Docker-compose from cisoassistant repo

**Comment by ab-smith** [comment]
Hey, any file size?

**Comment by schmitz-ens** [comment]
Hi, I had this problem with 33ko and 270ko file.

**Comment by nas-tabchiche** [comment]
Hi @schmitz-ens, thanks for the input! This is solved by #328  The issue will be closed once it is merged on main
